models/cfp.py

```diff
@@ -919,6 +924,33 @@ class LightningTalkProposal(Proposal):
     slide_link = db.Column(db.String, nullable=True)
     session = db.Column(db.String, default="fri")
 
+    should_be_exported = column_property(or_(
+        Proposal.is_accepted.expression,
+        Proposal.state == "new",
```

Comment:
Uninformed question: will this mean that any newly submitted other proposals would get exported or is that not what "new" actually means?